webpack.config.js

```diff
@@ -15,27 +13,63 @@ const mediaQueryAliases = {
   '(min-width: 1200px)': 'x-large-and-up',
 };
 
-const entryPoints = blockName => {
-  const tpl = {
-    [`${blockName}EditorScript`]: `./assets/src/blocks/${blockName}/${blockName}EditorScript.js`,
-    [`${blockName}EditorStyle`]: `./assets/src/styles/blocks/${blockName}/${blockName}EditorStyle.scss`,
-    [`${blockName}Script`]: `./assets/src/blocks/${blockName}/${blockName}Script.js`,
-    [`${blockName}Style`]: `./assets/src/styles/blocks/${blockName}/${blockName}Style.scss`,
-  };
-  const pathExists = ([,path]) => fs.existsSync(path);
-  return Object.fromEntries(Object.entries(tpl).filter(pathExists));
+const jsConfig = {
+  ...defaultConfig,
+  output: {
+    filename: '[name].js',
+    path: __dirname + '/assets/build'
+  },
+  optimization: {
+    minimizer: [
+      new TerserJSPlugin({}),
+    ]
+  },
 };
 
-module.exports = {
-  ...defaultConfig,
+const publicJsConfig = {
+  ...jsConfig,
+  resolve: {
+    alias: {
+      '@hooks': 'preact/hooks',
+      '@render': 'preact',
+      '@compat': 'preact/compat',
+    }
+  },
   entry: {
-    editorIndex: './assets/src/editorIndex.js',
     frontendIndex: './assets/src/frontendIndex.js',
     carouselHeaderFrontIndex: './assets/src/carouselHeaderFrontIndex.js',
+    AccordionScript: './assets/src/blocks/Accordion/AccordionScript.js',
```

Comment:
We can define a `const` for `blocks` instead of duplicating text. It could be `const BLOCKS_PATH='./assets/src/blocks/'`

Reply:
Idk, since a webpack config contains objects that are many lines long, it's actually harder to understand the file with the const being out of view most of the time.